core/templates/dev/head/components/OutcomeFeedbackEditorDirective.js

```diff
@@ -32,26 +33,7 @@ oppia.directive('outcomeFeedbackEditor', [
         };
 
         $scope.$on('saveOutcomeFeedbackDetails', function() {
-          // Remove null feedback. If the first element of the feedback is null
-          // or empty, clear the entire feedback array. This is so that if the
-          // first feedback is removed all feedback is thereby removed. Only
-          // first feedback is usable and editable. This also preserves all
-          // feedback entries after the first if the first is non-empty.
-          var nonemptyFeedback = [];
-          for (var i = 0; i < $scope.outcome.feedback.length; i++) {
-            var feedbackStr = $scope.outcome.feedback[i];
-            if (feedbackStr) {
-              feedbackStr = feedbackStr.trim();
-            }
-            if (feedbackStr) {
-              nonemptyFeedback.push(feedbackStr);
-            }
-            if (!feedbackStr && i === 0) {
-              // If the first feedback is empty, copy no more feedback after.
-              break;
-            }
-          }
-          $scope.outcome.feedback = nonemptyFeedback;
+
```

Comment:
Hm, interesting, do you even need this function at all then?

Reply:
removed